cdap-ui/app/index.html

```diff
@@ -34,7 +34,8 @@
           <div class="col-xs-12 col-md-6 text-uppercase">
             <p>
               <img src="/assets/img/cask.png" alt="" />
-              <span>&copy; 2015 Cask Data, Inc. All Rights Reserved.</span>
+              <span>Copyright &copy; 2014-<script type="text/javascript">today = new Date();document.write(today.getFullYear());</script>
```

Comment:
In angular world this should be done in a controller. 

The way we do it is, in main.js we have a ```BodyCtrl``` which will load the body element (or the home page). Right now it modifies the css class of the body based on the UI state. 

The ideal way to do this is,

Create a variable in the ```BodyCtrl``` and add it to the scope and use that variable here and use ng-bind-once which will bind that variable to this html element. That way we do javascript in js (controller) and bind the value to the DOM that gets updated only once when the app loads.